### ISSUE TITLE
WCAG 4.1.2: Ensures buttons have discernible text (.devsite-devprofile-button)

### ISSUE THREAD

**Issue by A11Ybot-OP** [firebase/friendlyeats-web#217]
#### Issue
`Ensures buttons have discernible text` ([button-name](https://accessibilityinsights.io/info-examples/web/button-name))
#### Target application
[Firebase](https://firebase.google.com/)
#### Element path
.devsite-devprofile-button
#### Snippet
`<button class="devsite-devprofile-button" data-tooltip="Google Developer Profile">`
#### How to fix
    Fix any of the following:
      Element does not have inner text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
      Element's default semantics were not overridden with role="none" or role="presentation"
#### Environment
Chrome version 98.0.4758.102
